src/content/operations/usage/_index.en.md

```diff
@@ -0,0 +1,666 @@
+    Type:                  Valid
+    Last Transition Time:  2020-11-30T19:59:44Z
+    Message:               Service was successfully synced to the broker
+    Reason:                
+    Status:                True
+    Type:                  Valid
+Events:                    <none>
+```
+<!-- markdownlint-enable no-trailing-spaces -->
+When the Service is exported successfully, it can be discovered as `nginx-ss.default.svc.clusterset.local` across the clusterset.
+In addition, the individual Pods can be accessed as `web-0.cluster-3.nginx-ss.default.svc.clusterset.local` and
+`web-1.cluster-3.nginx-ss.default.svc.clusterset.local`.
+
+##### 3. Test Service Discovery on **cluster-2**
+
+Verify that the exported `nginx-ss` Service was imported to **cluster-2**. Submariner (via Lighthouse) automatically creates a
+corresponding `ServiceImport`:
+
+```bash
+$ kubectl get -n submariner-operator serviceimport
+NAME                        TYPE           IP                AGE
+nginx-default-cluster3      ClusterSetIP   [100.3.220.176]   166m
+nginx-ss-default-cluster3   Headless                         5m48s
+```
+
+Next, run a test Pod on **cluster-2** and try to access the `nginx-ss` Service from within the Pod:
+
+```bash
+kubectl -n default  run --generator=run-pod/v1 \
+tmp-shell --rm -i --tty --image quay.io/submariner/nettest -- /bin/bash
+```
+<!-- markdownlint-disable no-hard-tabs -->
+```bash
+bash-5.0# dig nginx-ss.default.svc.clusterset.local
+
+; <<>> DiG 9.16.6 <<>> nginx-ss.default.svc.clusterset.local
+;; global options: +cmd
+;; Got answer:
+;; WARNING: .local is reserved for Multicast DNS
+;; You are currently testing what happens when an mDNS query is leaked to DNS
+;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 19729
+;; flags: qr aa rd; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 1
+;; WARNING: recursion requested but not available
+
+;; OPT PSEUDOSECTION:
+; EDNS: version: 0, flags:; udp: 4096
+; COOKIE: 0b17506cb2b4a93b (echoed)
+;; QUESTION SECTION:
+;nginx-ss.default.svc.clusterset.local. IN A
+
+;; ANSWER SECTION:
+nginx-ss.default.svc.clusterset.local. 5 IN A	10.3.0.5
+nginx-ss.default.svc.clusterset.local. 5 IN A	10.3.224.3
+
+;; Query time: 1 msec
+;; SERVER: 100.2.0.10#53(100.2.0.10)
+;; WHEN: Mon Nov 30 20:18:08 UTC 2020
+;; MSG SIZE  rcvd: 184
+
+bash-5.0#
+```
+
+You can also access the individual Pods:
+
+```bash
+bash-5.0# nslookup web-0.cluster3.nginx-ss.default.svc.clusterset.local
+Server:		100.2.0.10
+Address:	100.2.0.10#53
+
+Name:	web-0.cluster3.nginx-ss.default.svc.clusterset.local
+Address: 10.3.0.5
+
+bash-5.0# nslookup web-1.cluster3.nginx-ss.default.svc.clusterset.local
+Server:		100.2.0.10
+Address:	100.2.0.10#53
+
+Name:	web-1.cluster3.nginx-ss.default.svc.clusterset.local
+Address: 10.3.224.3
+
+bash-5.0#
+```
+<!-- markdownlint-enable no-hard-tabs -->
+#### Clean the Created Resources
+
+To remove the previously created Kubernetes resources, use the following commands on **cluster-3**:
+
```

Comment:
would it be easier to create the resources in a dedicated, new namespace? 
personally I find it easier to just delete the whole ns then, and it would demonstrate that the ns needs to be created in both clusters for it to work :)